src/platform_impl/windows/window_state.rs

```diff
@@ -79,6 +79,7 @@ bitflags! {
         const TRANSPARENT      = 1 << 6;
         const CHILD            = 1 << 7;
         const MAXIMIZED        = 1 << 8;
+        const MINIMIZABLE      = 1 << 12;
```

Comment:
This is a duplicate value (already in use for `MINIMIZED`, see a few lines down).

The highest existing number is 16 so I suggest using 17 here, then we can use 18 and 19 for `MAXIMIZABLE` and `CLOSABLE`.

Reply:
@caesar Oops. Yeah its a typo. 18 will work! Lets do that.

Reply:
Edited the bitflags as discussed